src/pages/index.js

```diff
@@ -168,10 +200,19 @@ function handleCardModalFormSubmit(evt) {
     name: cardModalNameInput.value,
     link: cardModalLinkInput.value,
   };
-  renderCard(inputValues);
-  evt.target.reset();
-  disableButton(cardModalSubmitButton, validationConfig);
-  closeModal(cardModal);
+
+  cardModalSubmitButton.textContent = "Saving...";
+  api.addNewCard(inputValues)
+    .then((newCard) => {
+      renderCard(newCard);
```

Comment:
* New cards should be added only with `prepend` to the **start of the section**, otherwise the user doesn’t see the new card (if there are **6 or more** cards on the page) and may think that it hasn’t been created

Please, add `"prepend"` as the 2nd argument:

```js

      renderCard(newCard, "prepend");

```